modules/tinymce/src/plugins/table/test/ts/browser/settings/TableToolbarTest.ts

```diff
@@ -18,6 +18,7 @@ describe('browser.tinymce.plugins.table.TableToolbarTest', () => {
 
   const pCreateEditor = (toolbar: string) => McEditor.pFromSettings<Editor>({
     plugins: 'table',
+    toolbar: 'tablecaption',
```

Comment:
Why was this added here? It looks like this wasn't reverted properly when the test logic was moved.

- [ ] Revert this change please